sdk/storage/Azure.Storage.Common/src/Shared/PartitionedUploader.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace Azure.Storage
 {
-    internal class PartitionedUploader<TServiceSpecificArgs, TCompleteUploadReturn>
+    internal class PartitionedUploader<TServiceSpecificData, TCompleteUploadReturn>
```

Comment:
What inspired the type rename?

Reply:
Look at the `PartitionedUploadBehaviors` created in `ShareFileClient`.

Shares has no "commit" function for partitioned uploads, but it needs to return something for `PartitionedUploader` to return to the user. Since the instance of that generic type is never altered by the uploader, only held and passed into each behavior delegate for them to interpret themselves, we hold onto the last received `UploadRange` response and store it in there. Then for the "commit," we just fetch it out of there and return it. Since we were no longer using the object to house just arguments, a rename felt like it would aid understanding.